web/app/styles/_colors.less

```diff
@@ -0,0 +1,19 @@
+:root
+{
+	@background: #FFFFFF; --background: @background;
+	@foreground: #141414; --foreground: @foreground;
+	@gray: #A9A9A9; --gray: @gray;
+
+
+	@primary: #0062C3; --primary: @primary;
+
+	@green: #1C820F; --green: @green;
```

Comment:
I'd name variables after the model, not the value. "Player1" and "Player2" are better IMO than "green" and "red"